modules/base/include/modules/base/processors/gridsystem.h

```diff
@@ -41,20 +41,18 @@
 namespace inviwo {
 
 namespace detail {
-template <unsigned N, typename T>
-constexpr T passthrough_the_value_helper(const T& t) {
-    return t;
-}
 
-template <typename T, unsigned... I>
-constexpr auto copyInitFilledArrayHelper(const T& t, std::index_sequence<I...>) {
-    return std::array<T, sizeof...(I)>{passthrough_the_value_helper<I>(t)...};
+template <typename T, std::size_t... Is>
+constexpr std::array<T, sizeof...(Is)> create_array(T value, std::index_sequence<Is...>) {
```

Comment:
we have util::make_array in stdextensions.h

Reply:
Nice, using the `util::make_array` made the code a bit cleaner. (Not using copy ctor, not changing the dispName/id afterwards)